ArduCopter/mode_auto.cpp

```diff
@@ -1072,13 +1138,19 @@ void Copter::ModeAuto::do_nav_wp(const AP_Mission::Mission_Command& cmd)
         }
     }
 
+    if(target_loc.alt!=current_loc.alt){
```

Comment:
Needs spaces:

- after if
- before and after `!=`
- after `)`

Reply:
revised and changed